webpack/assets/javascripts/react_app/components/common/forms/Form.stories.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { reduxForm } from 'redux-form';
+import { connect } from 'react-redux';
+
+import RadioButtonGroup from './RadioButtonGroup';
+import Form from './Form';
+import storeDecorator from '../../../../../../stories/storeDecorator';
+
+
+const formName = 'storybookForm';
+
+const StoryForm = () => {
+  const radios = [
+    {
+      label: 'Yes', value: 'yes',
+    },
+    {
+      label: 'No', value: 'no',
+    },
+    {
+      label: 'Do Not Know', value: 'dnk',
+    },
+  ];
+
+  return (
+    <Form>
+      <RadioButtonGroup name="steak" controlLabel="Would you like a steak?" radios={radios}/>
```

Comment:
can we change it to: would you like an hamburger? https://www.youtube.com/watch?v=lLuc6rtWkrM

Reply:
Changed steak to hamburger :)